src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -56,19 +62,48 @@ export default class ReplyField extends React.Component<Props, State> {
         prevEditorState = EditorState.forceSelection(prevEditorState, selectionState);
 
         this.state = {
+            activeItemIndex: 0,
             editorState: prevEditorState,
             popupReference: null,
         };
     }
 
     componentDidMount(): void {
         this.focusEditor();
+        // restore PopupList if it was active
+        this.updatePopupReference();
     }
 
     componentWillUnmount(): void {
         this.saveCursorPosition();
     }
 
+    getCollaborators = (activeMention: Mention | null): Collaborator[] => {
+        const { collaborators } = this.props;
+
+        if (!activeMention) {
+            return [];
+        }
+
+        const trimmedQuery = activeMention.mentionString.trim();
+        // fuzzySearch doesn't match anything if query length is less than 2
+        // Compared to empty list, full list has a better user experience
+        if (trimmedQuery.length < 2) {
+            return collaborators;
+        }
+
+        return collaborators.filter(({ item }) => {
+            if (!item) {
+                return false;
+            }
+
+            const isNameMatch = fuzzySearch(trimmedQuery, item.name, 0);
+            const isEmailMatch = 'email' in item && fuzzySearch(trimmedQuery, item?.email, 0);
```

Comment:
Is the optional chaining needed here? I would think that since your'e already checking the item on L94, this wouldn't be needed